website/content/docs/getting-started/index.mdx

```diff
@@ -0,0 +1,37 @@
+---
+layout: docs
+page_title: Getting Started with Waypoint
+description: |-
+  Install and run Waypoint with this five minute quick start tutorial.
+---
+# Waypoint Get Started
+
+The Waypoint Getting Started guides will help you become familiar with Waypoint, and learn how you can deploy applications with minimal friction through Waypoint.
+
+The tutorials are intended to be completed sequentially, as each one builds upon knowledge from the previous tutorial.
+You will learn how to install Waypoint, deploy an application, and remove applications through Waypoint.
+Most importantly, you will learn how to get an application from development to a target environment by using a single file, `waypoint.hcl`, and a single command `waypoint up`.
+
+![Files](/img/getting-started/intro-files.png)
+
+
+## Nomad
+
+To get started with Waypoint and deploy applications to a Nomad cluster, check out the [Getting Started with Nomad](/tutorials/get-started-nomad/get-started-intro).
+
+## Kubernetes
+
+Waypoint supports Kubernetes workflows and enables product teams to use existing Kubernetes configurations. Check out the [Getting Started with Kubernetes](tutorials/get-started-kubernetes/get-started-intro) to learn how you can use Waypoint with Kubernetes.
+
+
+## Docker
+
+You can deploy containerized workloads through Docker and Waypoint. Start deploying containerized applications by checking out the [Getting Started with Docker](/tutorials/get-started-docker/get-started-intro) tutorials.
+
+
+## Next Steps
+
+Afterwards, go in-depth with tutorials for AWS, Azure, GCP, Kubernetes, and
+Nomad [tutorials](https://learn.hashicorp.com/waypoint). For detailed
+Kubernetes-focused documentation, see the
```

Comment:
You were going to drop the "detailed kubernetes-focused documentation" line right?